src/pages/workspace/reportFields/WorkspaceListValuesPage.tsx

```diff
@@ -0,0 +1,271 @@
+                });
+            }
+
+            return (
+                <ButtonWithDropdownMenu
+                    onPress={() => null}
+                    shouldAlwaysShowDropdownMenu
+                    pressOnEnter
+                    buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
+                    customText={translate('workspace.common.selected', {selectedNumber: selectedValuesArray.length})}
+                    options={options}
+                    isSplitButton={false}
+                    style={[shouldUseNarrowLayout && styles.flexGrow1, shouldUseNarrowLayout && styles.mb3]}
+                />
+            );
+        }
+
+        return (
+            <View style={[styles.w100, styles.flexRow, styles.gap2, isSmallScreenWidth && styles.mb3]}>
+                <Button
+                    style={[isSmallScreenWidth && styles.flex1]}
+                    medium
+                    success
+                    icon={Expensicons.Plus}
+                    text={translate('workspace.reportFields.addValue')}
+                    onPress={() => Navigation.navigate(ROUTES.WORKSPACE_REPORT_FIELD_ADD_VALUE.getRoute(policyID))}
+                />
+            </View>
+        );
+    };
+
+    return (
+        <AccessOrNotFoundWrapper
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            policyID={policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_REPORT_FIELDS_ENABLED}
+        >
+            <ScreenWrapper
+                includeSafeAreaPaddingBottom={false}
+                style={[styles.defaultModalContainer]}
+                testID={WorkspaceListValuesPage.displayName}
+                shouldEnableMaxHeight
+            >
+                <HeaderWithBackButton
+                    title={translate('workspace.reportFields.listValues')}
+                    onBackButtonPress={Navigation.goBack}
+                >
+                    {getHeaderButtons()}
+                </HeaderWithBackButton>
+                <View style={[styles.ph5, styles.pb4]}>
+                    <Text style={StyleUtils.combineStyles([styles.sidebarLinkText, styles.optionAlternateText])}>{translate('workspace.reportFields.listInputSubtitle')}</Text>
+                </View>
+                {shouldShowEmptyState && (
+                    <WorkspaceEmptyStateSection
+                        containerStyle={[styles.bgTransparent, styles.mtAuto, styles.mbAuto]}
+                        title={translate('workspace.reportFields.emptyReportFieldsValues.title')}
+                        icon={Illustrations.EmptyStateExpenses}
+                        subtitle={translate('workspace.reportFields.emptyReportFieldsValues.subtitle')}
+                    />
+                )}
+                {!shouldShowEmptyState && (
+                    <SelectionList
+                        canSelectMultiple
+                        sections={[{data: valueList, isDisabled: false}]}
```

Comment:
Same, maybe we could make `valueList` return this structure on `useMemo` instead of defining a new array every time